.gitignore

```diff
@@ -164,3 +164,7 @@ Staticfile.auth
 *.ipr
 *.iws
 notebooks/*.html
+*.json
+google_searchconsole/parameters.py
```

Comment:
Where is this used? If it is secret values, there should be a template file that isn't ignored.

Reply:
created template file for upload